tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -1,736 +1,555 @@
 }
 
-setup_working_dir() {
-    local CONFIG_DIR=~/configs
-
-    local INTEL_KEY=$CONFIG_DIR/key_production.txt
-    local INTEL_SPID=$CONFIG_DIR/spid_production.txt
-
-    source_dir=$1
-    target_dir=$2
-
-    cd $source_dir || exit
-    ./litentry-worker signing-key | grep -oP '^Enclave account: \K.*$$' > enclave_account.txt
-    echo "Enclave account is prepared inside enclave_account.txt"
-
-    ENCLAVE_ACCOUNT=$(cat enclave_account.txt)
-    export ENCLAVE_ACCOUNT
-    echo "Enclave Account: $ENCLAVE_ACCOUNT"
-
-    optional=("key.txt" "spid.txt")
+function setup_working_dir {
+    echo "Setting up working dir ..."
+    cd "$ROOTDIR/tee-worker/bin" || exit
 
-    for file in "${optional[@]}"; do
-        source="${source_dir}/${file}"
-        target="${target_dir}/${file}"
-
-        if [ -f "$source" ]; then
-            cp "$source" "$target"
-        else
-            echo "$source does not exist, this is fine, but you can't perform remote attestation with this."
-        fi
-    done
-
-    for Item in 'enclave.signed.so' 'litentry-worker' 'aes_key_sealed.bin' 'ed25519_key_sealed.bin' 'enclave-shielding-pubkey.json' 'enclave-signing-pubkey.bin' 'rsa3072_key_sealed.bin' 'sidechain_db'; do
-      cp -r "${Item}" "${target_dir}"
-    done
-
-    # Only possible in TEE-Internal
-    cp $CONFIG "${target_dir}/mode_config.json"
-    if [ "$PRODUCTION" = true ]; then
-      cp $INTEL_KEY "${target_dir}/key_production.txt"
-      cp $INTEL_SPID "${target_dir}/spid_production.txt"
-    else
-      cp $INTEL_KEY "${target_dir}/key.txt"
-      cp $INTEL_SPID "${target_dir}/spid.txt"
-    fi
-}
-
-function restart_worker() {
-  # Need to make sure we have the JSON
-  cd $ROOTDIR || exit
-  worker_count=$(echo "$CONFIG" | jq '.workers | length')
-
-  for ((i = 0; i < worker_count; i++)); do
-    WORKER_DIR=/opt/worker/w${i}
-    # Remove previous logs if any
-    rm -r $ROOTDIR/tee-worker/log/worker${i}.log
-    # Prepare the Worker Directory before restarting
-    mkdir -p /opt/worker/w${i}
-    setup_working_dir $ROOTDIR/tee-worker/bin /opt/worker/w${i}
-
-    # We only need this in productive enclave
-    if [ "$PRODUCTION" = true ]; then
-      # Transfer balance to the enclave account that is generated
-      echo "Transferring balance to the enclave account"
-      cd $ROOTDIR/scripts/ts-utils/ || exit
-      yarn install
-      npx ts-node transfer.ts  $ENCLAVE_ACCOUNT
+    if [ "$PRODUCTION" = false ]; then
+      for f in 'key.txt' 'spid.txt'; do
+        [ -f "$f" ] || touch "$f"
+      done
     fi
 
-    cd $ROOTDIR/tee-worker || exit
-
-    source=$(echo "$CONFIG" | jq -r ".workers[$i].source")
-    flags=$(echo "$CONFIG" | jq -r ".workers[$i].flags[]")
-    subcommand_flags=$(echo "$CONFIG" | jq -r ".workers[$i].subcommand_flags[]")
+    for ((i = 0; i < WORKER_COUNT; i++)); do
+      worker_dir="$WORKER_BASEDIR/w$i"
+      mkdir -p "$worker_dir"
+      for f in 'key.txt' 'spid.txt' 'enclave.signed.so' 'litentry-worker'; do
```

Comment:
The `worker` will expect for `spid_production.txt` and `key_production.txt` in production mode.

Reply:
Do you know where can I get it :(

Reply:
I think we keep it inside the `configs` folder inside the server, can we have a flag to specify the path for it or we just assume that it was copied to the `bin` directory.

Reply:
Yes. Currently it's stored on `tee-staging` server.

Reply:
> I think we keep it inside the `configs` folder inside the server, can we have a flag to specify the path for it or we just assume that it was copied to the `bin` directory.

I'm not sure which one you 

> I think we keep it inside the `configs` folder inside the server, can we have a flag to specify the path for it or we just assume that it was copied to the `bin` directory.

I'm fine with whichever way you prefer.

Reply:
I think for now, let's assume we put in `bin` folder and the script copies from it.